internal/storage/storage.go

```diff
@@ -0,0 +1,56 @@
+package storage
+
+import (
+	"time"
+
+	t "github.com/offen/docker-volume-backup/internal/types"
+)
+
+// Interface for defining functions which all storage providers support.
```

Comment:
It would be great if all GoDoc comments follow the established style: https://go.dev/blog/godoc